CHANGELOG.md

```diff
@@ -32,6 +33,24 @@
 - [0.7.0](#070)
 - [0.6.0](#060)
 
+## 2.1.0
+
+### Core
+- :sunrise: **support ENV variable in configuration.** [#2743](https://github.com/apache/apisix/pull/2743)
+- :sunrise: **support TLS connection with etcd.** [#2548](https://github.com/apache/apisix/pull/2548)
+- generate create/update_time automatically. [#2740](https://github.com/apache/apisix/pull/2740)
+- add a deprecate log for enable_websocket in upstream.[#2691](https://github.com/apache/apisix/pull/2691)
+- refactor(consumer): remove id and use username only.[#2679](https://github.com/apache/apisix/pull/2679)
```

Comment:
Should we write it clear that people need to delete the obsolete id from the stored etcd data?

Reply:
added